tests/wpt/web-platform-tests/html/browsers/history/the-location-interface/location_reload.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>location_reload</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+  <body>
+    <div id="log"></div>
+
+    <iframe></iframe>
+
+    <script>
+
+    setup({explicit_done:true});
+
+    var url = new URL("./location_reload-iframe.html", window.location).href;
+
+    var iframe = document.querySelector("iframe");
+
+    var pingCount = 0;
+
+    // called from location_reload-iframe.html
+    window._ping = function(innerURL) {
+
+      pingCount++;
+
+      test(function() {
+        assert_equals(url, innerURL, "iframe url");
+      }, "iframe url should not change");
```

Comment:
Sorry I missed this last time. You're creating multiple tests with the same name, which would be painful if we failed one of them. Please just put everything in one `async_test`, like:
```js
async_test(function() {
  var url = new URL("./location_reload-iframe.html", window.location).href;
  ...
  window._ping = this.step_func(function() {
    ...
    assert_equals(url, innerURL, "iframe url");
    ...
    this.done();
  });

  iframe.src = url;
});